esphome/components/epaper_spi/epaper_spi_model_7p3in_spectra_e6.cpp

```diff
@@ -0,0 +1,42 @@
+#include "epaper_spi_model_7p3in_spectra_e6.h"
+
+namespace esphome::epaper_spi {
+
+static constexpr const char *const TAG = "epaper_spi.7.3in-spectra-e6";
+
+void EPaper7p3InSpectraE6::power_on() {
+  ESP_LOGI(TAG, "Power on the display");
```

Comment:
Can probably trim out "the display" given `TAG`
```suggestion
  ESP_LOGI(TAG, "Power on");
```